arm-billing/2017-02-27-preview/swagger/billing.json

```diff
@@ -0,0 +1,308 @@
+        "description": "Lists the available invoices for a subscritpion in reverse chronological order beginning with the most recent one.",
+        "externalDocs": {
+          "url": "https://go.microsoft.com/fwlink/?linkid=842057"
+        },
+        "x-ms-examples": {
+					"InvoicesList": {
+    				"$ref": "../examples/InvoicesList.json" 
+        	},
+					"InvoicesExpand": {
+    				"$ref": "../examples/InvoicesExpand.json" 
+        	}
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "$filter may be used to filter invoices by invoicePeriodEndDate. The filter supports 'eq', 'lt', 'gt', 'le', 'ge', and 'and'. It does not currently support 'ne', 'or', or 'not'"
+          },
+          {
+            "name": "$top",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "description": "$top may be used to limit the number of results to the most recent N invoices."
+          },
+          {
+            "name": "$skip",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "minimum": 1,
+            "description": "The number of items to skip over before returning elements."
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "$expand may be used to expand the downloadUrl property within a list of invoices. This enables download links to be generated for multiple invoices at once."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/InvoicesListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/invoices/{invoiceName}": {
+      "get": {
+        "tags": [
+          "Invoices"
+        ],
+        "operationId": "Invoices_Get",
+        "description": "Gets a named invoice resource. When getting a single invoice, the downloadUrl property is expanded automatically.",
+        "x-ms-examples": {
+					"InvoicesGet": {
+    				"$ref": "../examples/InvoicesGet.json" 
+        	}
+        },
+        "parameters": [
+          {
+            "name": "invoiceName", 
```

Comment:
try to be consistent in your parameter order (i.e. `subscriptionIdParameter` and `apiVersionParameter` in front vs. back)